.travis.yml

```diff
@@ -62,6 +58,11 @@ script:
       ./mvnw test $MAVEN_SKIP_CHECKS_AND_DOCS -B -pl $TEST_OTHER_MODULES
     fi
   - |
+    if [[ -v LDAP_TESTS ]]; then
+      presto-product-tests/bin/run_on_docker.sh \
+        singlenode-ldap -g ldap
```

Comment:
@electrum This should run the group `ldap_cli` instead. `ldap` group also includes some tests which needs LDAP support in JDBC driver (which is not yet there).